ci-operator/config/openshift/kubernetes-nmstate/openshift-kubernetes-nmstate-release-4.11.yaml

```diff
@@ -69,11 +69,11 @@ releases:
     candidate:
       product: ocp
       stream: nightly
-      version: "4.10"
+      version: "4.11"
   stable:
     release:
       channel: stable
-      version: "4.9"
+      version: "4.11"
```

Comment:
Should this be 4.10?

Reply:
We could set it to 4.10 as no stable 4.11 is available yet. But so we would need to update it in a few weeks again. But I am not very opinionated on this...

Reply:
Ah, okay. I was thinking this should match master, but I suppose master will get updated when 4.12 opens so it doesn't necessarily need to.